### PR TITLE
[Glance] Add communitize_image policy and image_community_admin role

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @Carthaca 

Regarding Jira task CCM-13726, please validate.

Regards,
Rajiv

### CODE REVIEW

File: openstack/glance/templates/etc/_glance-policy.json.tpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "context_is_cloud_admin":  "role:cloud_image_admin",
-    "context_is_admin":  "rule:context_is_cloud_admin",
+    "context_is_admin":  "rule:context_is_cloud_admin or role:admin or role:image_admin",
```
Comment (Carthaca) [inline openstack/glance/templates/etc/_glance-policy.json.tpl:3]: Why are you relaxing this rule?

Comment (rajivmucheli) [reply openstack/glance/templates/etc/_glance-policy.json.tpl:3]: role:admin or role:image_admin can also set images visibility to community, these roles neednt add role:image_community_admin again.

Comment (Carthaca) [reply openstack/glance/templates/etc/_glance-policy.json.tpl:3]: If I read the policy correctly they can't do this currently, rule `default` doesn't allow that.

Do you want to allow communitize_image for roles `admin` and `image_admin`, too? Please add it directly to that rule.

File: openstack/glance/templates/seeds.yaml [hunk 2]
```diff
@@ -29,6 +29,7 @@ spec:
   - name: cloud_image_admin
   - name: image_admin
   - name: image_viewer
+  - name: image_community_admin
```
Comment (Carthaca) [inline openstack/glance/templates/seeds.yaml]: Please remove the unused role here

File: openstack/glance/templates/etc/_glance-policy.json.tpl [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "get_images": "",
     "modify_image": "",
     "publicize_image": "rule:context_is_cloud_admin",
+    "communitize_image": "role:image_admin",
```
Comment (Carthaca) [inline openstack/glance/templates/etc/_glance-policy.json.tpl:12]: please add the permission for cloud_image_admin, e.g. `role:image_admin or rule:context_is_admin`

**Pull request merged**